1-estrategia/README.md

```diff
@@ -2,22 +2,129 @@
+
+__*¿Cómo puedo ganarme la confianza de estas personas por medio de mi producto?*__
+
+Por medio de las alianzas estratégicas con las ONG´s y la certificación de ellas, ya que desde nuestro sitio web se generan hipervínculos directos a los sitios web de esas organizaciones acreditadas y de igual manera conseguir que esas organizaciones nos mencionen en sus sitios. Por otro lado, los documentos y trámites disponibles en el sitio web deberán cumplir con los siguientes requerimientos: ser verídicos, formales, claros y de sitios oficiales de las instituciones públicas del País Vasco y de reconocidas organizaciones no gubernamentales.
+
+
 ### Circunstancias de uso
 
+__Quien__:
+
+El público al que nos dirigimos son migrantes de entre 20 y 45 años que llegan al País Vasco. Nos hemos centrado en esta franja de edad, ya que según una encuesta realizada por el Gobierno Vasco en el año 2010 el 47% de los inmigrantes que llegan tienen entre 25 y 44 años, un 34% son menores de 25 años y tan solo el 19% son mayores de 45 años. Por esta razón, hemos concretado nuestro publico objetivo en este rango de edad, ya que además son las personas que más uso hacen de plataformas web, redes sociales y aplicaciones para móvil o tablets.
+
+__Que__: 
+
+La plataforma web de Aupa , informará a las personas migrantes que se encuentran en el País Vasco. Es una plataforma creada con el fin de que las personas afectadas encuentren ayudas administrativas, refugio, educación, alimentación, asistencia sanitaria o que puedan incorporarse a organizaciones que defiendan sus derechos, como ACNUR u otras ONGs.
+
+__Cuando__: 
+
+Desde el momento en que llegan al País Vasco para instalarse, debido a que la plataforma se centra en ayudar a acoplarse en la nueva cultura y también en la reubicación para que así puedan satisfacer sus necesidades básicas. 
+
+__Dónde__: 
+
+Pais Vasco. 
+
+__Para que__: 
+
+Facilitar la integración de los inmigrantes en la comunidad Vasca, brindarles esperanza y que puedan superar la crisis en la que se ven sumergidos.
+
+
 ### Criterios de diseño
 
+Aupa tiene como fin comunicar e informar a los migrantes del País Vasco, por ende los criterios de diseño los definimos de la siguiente manera: 
+
+* Accesibilidad: El sitio web está pensado para obtener acceso multipantalla, es decir, nuestros usuarios pueden ingresar al sitio web desde la computadora, la tablet o el teléfono móvil. 
+
+* Findability:  En nuestro sitio web, la intuitividad es muy importante, las pestañas deben ser muy claras y con una organización fácil para los inmigrantes. 
+
+* Funcionalidad: se refiere a que todo lo disponible en la web funcione correctamente. 
```

Comment:
Si no tenéis nada específico que definir (como en este caso), no hace falta que lo incluyáis como criterio de diseño. Efectivamente, "Funcionalidad" _se refiere a_ esto que decís, ¿pero en qué se traduce esto en vuestro sitio?